app/src/main/java/rightmesh/left/io/gpstracker/MainActivity.java

```diff
@@ -27,14 +27,15 @@
  * An activity that listens to GPS updates and reports them back to the RightMesh SuperPeer.
  */
 public class MainActivity extends AppCompatActivity {
-    private static final String TAG = MainActivity.class.getCanonicalName();
+    static final String TAG = MainActivity.class.getCanonicalName();
```

Comment:
Does this need to be only package private or would it be better to keep it private?

Reply:
As a PMD suggestion, it would better to use package private for `TAG`. You can take a look at the PMD warning below. 

![screen shot 2019-02-28 at 10 50 20 am](https://user-images.githubusercontent.com/8343870/53590526-cc216180-3b46-11e9-915e-cdb81169a21b.png)

In `MainActivity`, I need to use `TAG` in some callbacks so that violate PMD rules if I used `private` for it.